graphql/types/userInputs.ts

```diff
@@ -52,7 +52,22 @@ export class RegistrationSchool {
 }
 
 @InputType()
-export class RegisterPupilInput implements RegisterPupilData {
+class RegisterPupilInputShared {
```

Comment:
Hm, I do have some accounts at IDPs with bogus names and might want to change the name when I register at Lern-Fair. Maybe we should return name + email, then allow the user to change them? 

If we allow that, it might be easier to use the existing registerPupil/Student, and just pick up IDP info stored in the session

Reply:
Also e.g. if parents sign on, they might want to change the name to their child?